recipients_app/.vscode/settings.json

```diff
@@ -1,3 +1,9 @@
 {
-	"java.configuration.updateBuildConfiguration": "automatic"
+	"java.configuration.updateBuildConfiguration": "automatic",
+	"dart.lineLength": 120,
+	"[dart]": {
+    "editor.rulers": [
+        120
+    ],
+}
```

Comment:
Set max line length for dart files to 120 to ensure all developers formatting with this rule. This avoid unnecessary changes in pull requests.